identity.go

```diff
@@ -502,6 +503,16 @@ func LoadIdentity(cfg Config) (Identity, error) {
 				return nil, err
 			}
 
+			if strings.EqualFold("true", os.Getenv("ZT_DEBUG_CERTS")) {
```

Comment:
Is this some standard we are going with now for debug logging?

Reply:
It's a way to allow debugging for a specific section. I don't know if it's the best thing. I could have also logged it at trace, but that would have turned on many other things. Could have used logging channels, but we don't support setting those at startup. The main problem I see with this approach is that it's not something you can figure out from the outside.

I could leave it, remove it or change it to something better if you've got suggestions. Thoughts?